_data/contributors.json

```diff
@@ -388,6 +388,19 @@
             "familyName": "Chen",
             "givenName": "July"
         },
+        {
+            "@id": "https://www.openmicroscopy.org/contributors#NicoChiaruttini",
+            "@type": "Person",
+            "affiliation": [
+                {
+                    "@type": "Organization",
+                    "name": "EPFL",
+                    "url": "https://www.epfl.ch/"
+                }
+            ],
+            "familyName": "Chiaruttini",
+            "givenName": "Nicolas"
```

Comment:
As a minor RFE this could include a 

```
            "sameAs": "https://github.com/NicoKiaru"
```

line so that the contributors page links to the appropriate GitHub profile.